index.bs

```diff
@@ -896,15 +850,108 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
         </p>
       </section>
 
-      <section>
-        <h4 id='is-encrypted-decode-supported'>
-          <dfn>Check Encrypted Decoding Support</dfn>
-        </h4>
+      <section algorithm>
+        <h4 id="check-mime-type-support"><dfn abstract-op noexport lt="Check MIME Type Support|check MIME type support">Check MIME Type Support</dfn></h4>
+        <p>
+          To [$check MIME type support$], given a {{DOMString}} |contentType|, a
+          string |media|, and either a {{MediaEncodingType}} or
+          {{MediaDecodingType}} |encodingOrDecodingType|, perform the following
+          steps. They return either a boolean that indicates whether
+          the MIME type of |contentType| is [=supported by the user agent=], or
+          <code>failure</code> if |contentType| failed to be parsed:
```

Comment:
Since there are three possible outcomes for these steps would an enum-like return value be better?

```suggestion
          steps. They return <code>supported</code> if
          the MIME type of |contentType| is [=supported by the user agent=], <code>unsupported</code>
          if |contentType| is not supported, or
          <code>failure</code> if |contentType| failed to be parsed:
```

Reply:
Yes, this would work better.